### PR TITLE
[14_0_X] Script to test MC production (UL-like)

### PR DESCRIPTION
**Pull request by AdrianoDee**
Plain backport of #44578

### CODE REVIEW

**Comment by AdrianoDee** [comment]
please test

**Comment by AdrianoDee** [comment]
assigne core
(as in master)

File: Configuration/PyReleaseValidation/test/BuildFile.xml [hunk 1]
```diff
@@ -2,3 +2,29 @@
...
+    The conditions used here for 2022 and 2023 will be changed 
+    to use the proper GT from ALCA including the BS smearing 
+    record from GT. For the moment we use the automatic ones.
+    By default these tests won't fail if a failure happens at
+    HLT step. If you want to catch those errors please set the
+    environment variable CMSSW_MC_SETUP_TEST_CATCH_HLT.
+-->
+
+<!-- In CMSSW_13_0_18 the auto:phase1_2023_realistic (pre BPix) is 130X_mcRun3_2023_realistic_postBPix_v1 -->
+<test name="test_MC_23_setup" command="test_mc_setup/test_MC_setup.sh auto:phase1_2023_realistic Run3_2023 2023v12 CMSSW_13_0_18 130X_mcRun3_2023_realistic_v13 Realistic25ns13p6TeVEarly2023Collision" />
```
Comment (makortel) [inline Configuration/PyReleaseValidation/test/BuildFile.xml:17]: Just for my understanding, would the (14_0_X) GT change come later?

**Comment by makortel** [comment]
Comparisons show the usual
* https://github.com/cms-sw/cmssw/issues/39803
* https://github.com/cms-sw/cmssw/issues/43293
* https://github.com/cms-sw/cmssw/issues/43790

**Comment by makortel** [comment]
+core

**Comment by AdrianoDee** [comment]
+pdmv

**Comment by srimanob** [comment]
+Upgrade

**Comment by rappoccio** [comment]
+1

**Pull request merged**